### PR TITLE
Update display font of Memory Browser to monospace

### PR DESCRIPTION
**Pull request by quyettrinhrvc**

### CODE REVIEW

**Comment by jonahgraham** [comment]
Is `Consolas` available everywhere, I see some people recommend having monospace and others as a fallback, maybe something like:

```css
font-family: Consolas, "Courier New", Courier, monospace;
```

**Comment by jonahgraham** [comment]
Hi @quyettrinhrvc - an additional thing - you need to create an eclipse.org account so that the bot can identify that you are a Renesas employee covered by that legal agreement. Create an account at https://accounts.eclipse.org/ and once created enter your GitHub user id in https://accounts.eclipse.org/user/edit

**Comment by quyettrinhrvc** [comment]
Hi @jonahgraham ,
I updated CSS file for font fixing, and as my checking, it can work for both Windows and Linux now.
Could you please help me review it?
About eclipse.org account, I am contacting our IT team to create it. I have no right to create it by myself.
Sorry for this inconvenience!

**Comment by jonahgraham** [comment]
OK. We'll wait, I hope it doesn't take long, but it is worth the investment of time now so you can contribute more easily in the future.

Cc: @Kummallinen

**Comment by quyettrinhrvc** [comment]
Hi @jonahgraham ,
Sorry for my lateness.
FYI, RVC IT finished approval for email domain and I can validate eclipse ECA now.

File: src/memory/client/MemoryBrowser.tsx [hunk 1]
```diff
@@ -142,7 +142,7 @@ export class MemoryBrowser extends React.Component<Props, State> {
             size={6}
             title="Number of bytes to fetch, in decimal or hexadecimal"
             defaultValue={this.lengthReq}
-            onChange={(event) => (this.addressReq = event.target.value)}
+            onChange={(event) => (this.lengthReq = event.target.value)}
```
Comment (jonahgraham) [inline src/memory/client/MemoryBrowser.tsx:145]: This line seems unrelated to the rest of the change. Please provide it as a separate PR with an explanation.

**Comment by jonahgraham** [comment]
> Hi @jonahgraham , Sorry for my lateness. FYI, RVC IT finished approval for email domain and I can validate ECA now.

Excellent - thanks @quyettrinhrvc for getting that resolved.

File: src/memory/client/MemoryBrowser.tsx (hunk 1 above)
Comment (quyettrinhrvc) [reply src/memory/client/MemoryBrowser.tsx]: I reversed this point

**Review by jonahgraham — APPROVED** [review]
